crypto/nextgen_crypto/src/bls12381.rs

```diff
@@ -149,16 +137,16 @@ impl TryFrom<&[u8]> for BLS12381PrivateKey {
     }
 }
 impl ValidKey for BLS12381PrivateKey {
-    // TODO(ladi): implement!
     fn to_bytes(&self) -> Vec<u8> {
-        unimplemented!("ask ladi!")
+        // This is expected to succeed as we just return the bytes of an existing key.
+        bincode::serialize(&self.0).unwrap()
     }
 }
 
 impl Genesis for BLS12381PrivateKey {
     fn genesis() -> Self {
-        let mut buf = [0u8; threshold_crypto::PK_SIZE];
-        buf[threshold_crypto::PK_SIZE - 1] = 1;
+        let mut buf = [0u8; BLS12381_PUBLIC_KEY_LENGTH];
```

Comment:
Genesis returns the private key, not the public key right?

Reply:
Nice catch, oops it originally mentioned `PK_SIZE`, hm,  this is a fix then. Thanks!